src/router.ts

```diff
@@ -328,3 +330,22 @@ export const router = createRouter({
   history: createWebHistory(config.ADMIN_GUI_PATH),
   routes,
 })
+
+const extractRouteName = (route) => {
+  const name = route.name
+  let routeName = '/'
+
+  if (typeof (name) === 'string') {
+    routeName = name.toString()
+  }
+
+  if (typeof (name) === 'symbol') {
+    routeName = Symbol.keyFor(name) || '/'
```

Comment:
`keyFor` accepts a string for a symbol name and re-grab it again. It should not be used at here.